docs/extend/modular-resources/_index.md

```diff
@@ -35,7 +35,13 @@ A module can provide one or more modular resources, and can be added to your rob
 ## The Viam Registry
 
 The [Viam registry](https://app.viam.com/registry) allows hardware and software engineers to collaborate on their robotics projects by writing and sharing custom modules with each other.
-You can add a module from the Viam registry directly from your robot's **Configuration** tab in [the Viam app](https://app.viam.com/), using the **+ Create component** button.
+This collaboration is facilitated by modules that can be written in a variety of languages (Go, Python, C++, Rust, etc.) and reliable [underlying APIs](/program/).
+
+You can add your modules to the registry, whether you created them manually or through continuous integration, and dictate the module's version and update strategy.
```

Comment:
Ah, I had been referencing [Eliot's blog post](https://www.viam.com/post/announcing-the-viam-modular-registry) announcing the registry where he talked about continuous integration as an option for publishing modules, but I may have misunderstood what he meant by upgrade policy. "You can control the module version and upgrade policy. Module upgrades are deployed automatically to your machine." is what I had read

Reply:
I think what you can do is specify a specific module version and then it won't automatically upgrade. @andf-viam do you know more?

Reply:
Yes, that's correct. You can pin to patch (UI: **Patch (X.Y.Z)** in the App) to prevent all upgrades. [This section ](https://docs.viam.com/modular-resources/configure/#configure-version-update-management-for-a-registry-module) attempts to cover that. "Upgrade policy" probably refers to that: pinning to specific (no upgrades), limiting to major or minor (just upgrade to family), or latest (no restrictions, updates asap when avail).

Reply:
Just also found https://docs.viam.com/modular-resources/configure/#configure-version-update-management-for-a-registry-module